demo/src/app/components/+pagination/demos/disable-direction-links/disable-direction-links.html

```diff
@@ -0,0 +1 @@
+<pagination [directionLinks]="false" [totalItems]="55"></pagination>
```

Comment:
Demos like this one should have a button that toggles the part of the functionality that this demo shows.
`<button (click)="directionLinks = !directionLinks">azaza</button>`